Find a functionally equivalent code:com.android.internal.policy.impl.PhoneWindowManager.areTranslucentBarsAllowed:COMMENT
Method Modifier: private     internal    
Comment:/**
 * @return whether the navigation or status bar can be made translucent
 *
 * This should return true unless touch exploration is not enabled or
 * R.boolean.config_enableTranslucentDecor is false.
 */

@@ -1,3 +1,3 @@
 {
-    return mTranslucentDecorEnabled && !mTouchExplorationEnabled;
+    return mTranslucentDecorEnabled && !mAccessibilityManager.isTouchExplorationEnabled();
 }

